nservicebus/upgrades/5to6-moving-away-from-ibus.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Moving away from `IBus` in NServiceBus Version 6
+summary: Describes how to send messages without the IBus
+---
+
+Starting with Versions 6 and above, the `IBus` interface has been deprecated.
+
+The `IBus` interface was one of the key interfaces when using previous versions of NServiceBus. It provided access to many of the important operations like sending messages, subscribing to events, and manipulating headers. The `IBus` interface was injected into the container using dependency injection and was accessible in all the message handlers and other injected components in the container as well. 
```

Comment:
> The `IBus` interface was injected into the container using dependency injection and was accessible in all the message handlers and other injected components in the container as well.  

has 2 ands, hard to grasp. Perhaps a scaled down version?

The `IBus` was available through dependency injection in message handlers and custom components.